soa/types/time.h

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
We tend to put a header in every header and source file with the file_name, author, date and a little description of what the header is actually doing.

I encourage you to do the same